solutions/usaco-899.mdx

```diff
@@ -0,0 +1,114 @@
+---
+id: usaco-899
+source: USACO Gold 2019 January
+title: Shortcut
+author: Ryan Chou
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_shortcut_gold_jan19.html)
+
+## Explanation
+
+A simple brute-force approach would be to find the shortest path for all of the nodes, and then try to add a shortcut from $1$ to $i$ and find the shortest path again for all $i$.
+
+Using this approach, the answer is $\min(\texttt{dist}-\texttt{short}_i)$ for all i, where $\texttt{dist}$ is the travel time that all cows take without a shortcut and $\texttt{short}_i$ represents the travel time that all cows take when there is a shortcut from $1$ to $i$. Although this will certainly TLE, it hints at a faster way to compute the time saved. 
```

Comment:
```suggestion
Using this approach, the answer is $\min(\texttt{dist}-\texttt{short}_i)$ for all $i$, where $\texttt{dist}$ is the travel time that all cows take without a shortcut and $\texttt{short}_i$ represents the travel time that all cows take when there is a shortcut from $1$ to $i$. Although this will certainly TLE, it hints at a faster way to compute the time saved. 
```